src/components/PrimarySideBar.test.tsx

```diff
@@ -1,32 +1,42 @@
 import { render, screen, userEvent } from '../utils/test-utils';
 import { PrimarySideBar } from './PrimarySideBar';
 
+const noop = () => {
+  // Empty
+};
+
```

Comment:
You can use the `noop` function from the utils file

Reply:
You are right! Fixed.